examples/github_app.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+r"""Minimal Flask application example for development with github handler.
+
+Usage:
+
+1. Register a gihub application with `Authorization callback URL` as
+   `http://localhost:5000/oauth/authorized/github/`
+
+2. Ensure you have ``github3.py`` package installed:
+
+   .. code-block:: console
+
+      cdvirtualenv src/invenio-oauthclient
+      pip install -e .[github]
+
+3. Grab the *Client ID* and *Client Secret* after registering the application
+   and add them to your instance configuration as `consumer_key` and
+   `consumer_secret`.
```

Comment:
Maybe keeping them as OS variables:
```console
$ export GITHUB_APP_CREDENTIALS_KEY=...
$ export GITHUB_APP_CREDENTIALS_SECRET=...
```

or `github.json`:
```json
{"consumer_key": "...", "secret_key": "..."}
```
and later `GITHUB_APP_CREDENTIALS=json.loads('github.json')`?